stage1/init/init.go

```diff
@@ -185,30 +185,17 @@ func stage1() int {
 	}
 
 	mirrorLocalZoneInfo(c.Root)
-	c.MetadataServiceURL = common.MetadataServicePublicURL()
```

Comment:
@jonboulle Non private-net case runs without registering with metadata svc. Per spec, this technically makes it a non-compliant ACE.